README.md

```diff
@@ -187,6 +189,7 @@ file, `config.json`, is as follows.
       "build-systems" : [ "pnc", "koji" ],
       "cache-lifespan" : 3600000,
       "cache-max-idle" : 3600000,
+      "cache-location" : "/home/user/.buildfinder/cache"
```

Comment:
I'd rather we make the option to set `CONFIG_PATH` instead and leave the cache hardcoded relative to that directory.

Also, I would reference `${user.home}` in the docs, or however it already has it instead of hardcoding a value.